frontend/src/metabase/common/components/DataPicker/components/DataPickerModal.tsx

```diff
@@ -83,19 +87,35 @@ export const DataPickerModal = ({
     databaseId,
   });
 
-  const { tryLogRecentItem } = useLogRecentItem();
+  const QUESITON_PICKER_MODEL_FILTER_OPTIONS = useMemo(() => {
```

Comment:
Since this is no longer a constant (across renders), maybe `questionPickerModelFilterOptions`?